lib/base/model.js

```diff
@@ -485,7 +485,7 @@ ModelBase.prototype.isNew = function() {
  * @returns {Object} Serialized model as a plain object.
  */
 ModelBase.prototype.serialize = function(options) {
-  if (!options) options = {};
+  if (!_.isPlainObject(options)) options = {};
```

Comment:
If you could use something that doesn't depend on lodash that would be much appreciated. It's not about performance but more of not relying on third party code if not really required.

Using `typeof` with additional check for `null` is fine like in the example you linked to in the PR description.

Reply:
The problem with that kind of check is, that it returns true for array:
```shell
> typeof [] === 'object'
true
```

That might not be a problem as properties can be assigned to array too:
```javascript
let arr = [];
arr.visibility = true;
```

So if you are fine with it, I can change it?

Reply:
According to the spec of `JSON.stringify` this method will never be called with an array, so a user would have to be doing something really weird for that happen, in which case setting a `visibility` property on that array would be the least of the user's problems it seems.

I think it's fine.

Reply:
Alright. I will update this then.